contrib/test_jenkins.sh

```diff
@@ -408,6 +408,57 @@ run_uct_hello() {
 	rm -f ./uct_hello_world
 }
 
+run_client_server() {
+    dev=$1
+
+    test_name=ucp_client_server
+
+    if [ ! -x ${test_name} ]
+    then
+        gcc -o ${test_name} ${ucx_inst}/share/ucx/examples/${test_name}.c \
+        -lucp -lucs -I${ucx_inst}/include -L${ucx_inst}/lib \
+        -Wl,-rpath=${ucx_inst}/lib
+    fi
+
+    iface=`ibdev2netdev | grep Up | awk '{print $5}' | head -1`
+    server_ip=`ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}'`
+
+    # run server side
+    UCX_NET_DEVICES=${dev} UCX_TLS=rc ./${test_name} &
```

Comment:
need to use random tcp port, see run_hello(), or multiple execution of the test on same machine will fail